### PR TITLE
Added new response type

### PR DESCRIPTION
**Pull request by VaghinakDev**

### CODE REVIEW

**Review by dshabin — APPROVED** [review]

File: src/superannotate/lib/core/service_types.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+from pydantic import Extra
+
+
+class UserLimits(BaseModel):
+    super_user_limit: int
+    project_limit: int
+    folder_limit: int
+
+    def has_enough_slots(self, count: int):
+        if count > self.super_user_limit:
```
Comment (davitavagyan) [inline src/superannotate/lib/core/service_types.py:17]: The order of print seems to be incorrect, should be folder, project and super-admin

File: src/superannotate/lib/core/usecases.py [hunk 2]
```diff
@@ -3249,56 +3249,57 @@ def execute(self):
...
-                Key=auth_data["images"][str(image_data["id"])]["annotation_json_path"],
-                Body=json.dumps(self._annotations),
-            )
-            if (
-                self._project.project_type == constances.ProjectType.PIXEL.value
-                and os.path.exists(self._annotation_path)
-            ):
-                with open(self._annotation_path, "rb") as descriptor:
-                    file = io.BytesIO(descriptor.read())
+            if response.ok:
```
Comment (davitavagyan) [inline src/superannotate/lib/core/usecases.py:3258]: If not response.ok:
   return

File: src/superannotate/lib/core/usecases.py [hunk 3]
```diff
@@ -3249,56 +3249,57 @@ def execute(self):
...
+                        )
+                if self._verbose:
+                    logger.info(
+                        "Uploading annotations for image %s in project %s.",
+                        str(image_data["name"]),
+                        self._project.name,
+                    )
+
+                if (
+                    self._project.project_type == constances.ProjectType.PIXEL.value
```
Comment (davitavagyan) [inline src/superannotate/lib/core/usecases.py:3292]: This can be in line 3273

File: src/superannotate/lib/core/service_types.py (hunk 1 above)
Comment (VaghinakDev) [reply src/superannotate/lib/core/service_types.py:17]: This logic will be moved to the usecases

File: src/superannotate/lib/core/usecases.py (hunk 3 above)
Comment (VaghinakDev) [reply src/superannotate/lib/core/usecases.py]: This very confusing case, thanks updated

File: src/superannotate/lib/core/usecases.py (hunk 2 above)
Comment (VaghinakDev) [reply src/superannotate/lib/core/usecases.py:3258]: This is something like a rule of use case def
`def execute(): `
`    if self._is_valid():`
          code_block
          response.data  = ...
      return response
`

**Pull request merged**